src/app/switcher/switcher.service.ts

```diff
@@ -129,6 +188,9 @@ export class SwitcherService {
   }
 
   switchProgram(programObj: ProgramObj): Observable<any> {
+    // initialise Pusher and apollo here if there stack info in storage
+    this.sharedService.initPusherApollo();
```

Comment:
Hi @sasangachathumal I prefer to call this "initWebServices" so we can reuse that for different services in the future.

Reply:
sure. will change the naming.